terraform/modules/task-definitions/content-store/variables.tf

```diff
@@ -19,6 +19,10 @@ variable "mongodb_url" {
   type = string
 }
 
+variable "plek_hostname_prefix" {
```

Comment:
This would probably benefit from a description mentioning that it's primarily for prepending the "draft-" to the names of its dependencies.

In the long run we might be better off not using this feature of Plek and just overriding the specific dependencies which need to change, rather than having Plek prepend "draft-" to everything and then having to specify exceptions to that.

Reply:
Thanks, I've made this name more specific.